docs/light/getting_started.md

```diff
@@ -9,12 +9,12 @@ To start a REST server, we need to specify the following parameters:
 | trust-node  | bool      | "false"                 | true     | Whether this LCD is connected to a trusted full node |
 | trust-store | DIRECTORY | "$HOME/.lcd"            | false    | directory for save checkpoints and validator sets    |
```

Comment:
Note: if you click View, you’ll see that the table is not being formatted as such @zramsay

Reply:
fixed